Doc/library/functions.rst

```diff
@@ -1562,13 +1562,19 @@ are always available.  They are listed here in alphabetical order.
    .. versionchanged:: 3.11
       The ``'U'`` mode has been removed.
 
-.. function:: ord(c)
+.. function:: ord(character, /)
 
-   Given a string representing one Unicode character, return an integer
-   representing the Unicode code point of that character.  For example,
+   Return the ordinal value of a character.
```

Comment:
"The ordinal value" is from the description of the original `ord()` function in Pascal. It explains the name of the function and can help to mnemonize it.